packages/serverless-contracts/src/contracts/apiGateway/features/openApiDocumentation.ts

```diff
@@ -28,7 +29,9 @@ export const getOpenApiDocumentation = <
           // This cast is done because there is differences between JsonSchema and OpenAPIV3.SchemaObject specs
           // It may be fixed later
           // @ref https://swagger.io/specification/
-          schema: contract.outputSchema as OpenAPIV3.SchemaObject,
+          schema: contract.outputSchemas[
+            HttpStatusCodes.OK
+          ] as OpenAPIV3.SchemaObject,
```

Comment:
Add a comment here for improvement: given this PR, we will be able to be much more precise on the openAPI schema 🚀